### DISCUSSION TITLE
CODEOWNERS

### DISCUSSION THREAD

**Discussion by whippingdot** [whippingdot/Language-Tutorials#18] [Add Page]
Hey, can I add a CODEOWNERS file to the repo? For now I am doing it but tell me if you want me to delete it.

**Comment by darkdarcool** [comment]
@whippingdot wdym

**Reply by whippingdot** [reply]
Nothing, I already added it

**Reply by darkdarcool** [reply]
ah ok

**Reply by kokonut27** [reply]
huh

**Reply by darkdarcool** [reply]
> huh

He means like he is giving credit for the people that work on the tutorials

**Reply by whippingdot** [reply]
nope a codeowners file makes a person a must-need reviewer in a PR if the file is their's. I made the .cookeylang files or wtv files @cursorweb's, and so on...

**Reply by darkdarcool** [reply]
@whippingdot oh ok
